portfolio/components/ui/Figures/FinancialMathOverlay/FinancialMathOverlay.module.css

```
@@ -81,6 +81,21 @@
   pointer-events: none;
 }
 
+/* Receipt type badge */
+.receiptTypeBadge {
+  font-family: "Monaco", "Menlo", "Consolas", monospace;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)